### DISCUSSION TITLE
Bestbuy time out 3000ms with VPN

### DISCUSSION THREAD

**Discussion by killercam1020** [jef/streetmerchant#964] [Help]
### Description

 AUTO_ADD_TO_CART="true"
COUNTRY="usa"
LOG_LEVEL="info"
OPEN_BROWSER="true"
SHOW_ONLY_BRANDS= sony,microsoft
SHOW_ONLY_MODELS= ps5 console,ps5 digital,xbox series x
SHOW_ONLY_SERIES="sonyps5c,sonyps5de,xboxsx"
STORES="bestbuy,walmart,amazon,target,newegg,playstation,amazon-uk,amazon-de"
LOW_BANDWIDTH="true"
PAGE_SLEEP_MIN=2000
PAGE_SLEEP_MAX=3000

**Comment by killercam1020** [comment]
navigation timeout of 30000 ms exceeded is the exact message

**Comment by peacepenguin** [comment]
What vpn provider and software do you use?

**Comment by maglat** [comment]
I'm getting timeout even without VPN. But only when I run it on my Unbuntu server or raspberry pi. On my Mac, it works.

Maybe the shops detect the OS and block Ubtuntu Server automatically

**Comment by sassquatch-cloud** [comment]
You might try setting `USER_AGENT=` to a typical agent so it looks like you are a normal computer, and personally I have found that bestbuy doesn't seem to like low bandwidth mode so I have it turned off. 

As a quick fix you can set `PAGE_TIMEOUT=0` and bestbuy has not given me errors since, but idk if that might effect anything else.

**Comment by riskynacho** [comment]
> You might try setting `USER_AGENT=` to a typical agent so it looks like you are a normal computer, and personally I have found that bestbuy doesn't seem to like low bandwidth mode so I have it turned off.
> 
> As a quick fix you can set `PAGE_TIMEOUT=0` and bestbuy has not given me errors since, but idk if that might effect anything else.

Just fyi, the default agent is `Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36` if you leave it blank.
